diff --git a/build/npm/postinstall.ts b/build/npm/postinstall.ts
index 0dd0b514ecf..b9cf667ec91 100644
--- a/build/npm/postinstall.ts
+++ b/build/npm/postinstall.ts
@@ -60,7 +60,7 @@ async function npmInstallAsync(dir: string, opts?: child_process.SpawnOptions):
 		shell: true,
 	};
 
-	const command = process.env['npm_command'] || 'install';
+	const command = process.env['CHATERO_NPM_COMMAND'] || process.env['npm_command'] || 'install';
 
 	if (process.env['VSCODE_REMOTE_DEPENDENCIES_CONTAINER_NAME'] && /^(.build\/distro\/npm\/)?remote$/.test(dir)) {
 		const syncOpts: child_process.SpawnSyncOptions = {
