diff --git a/src/vs/workbench/contrib/inlineCompletions/browser/inlineCompletionLanguageStatusBarContribution.ts b/src/vs/workbench/contrib/inlineCompletions/browser/inlineCompletionLanguageStatusBarContribution.ts
--- a/src/vs/workbench/contrib/inlineCompletions/browser/inlineCompletionLanguageStatusBarContribution.ts
+++ b/src/vs/workbench/contrib/inlineCompletions/browser/inlineCompletionLanguageStatusBarContribution.ts
@@ -69,9 +69,9 @@ export class InlineCompletionLanguageStatusBarContribution extends Disposable im
 
 			const statusMap: Record<typeof status, { shortLabel: string; label: string; loading: boolean }> = {
 				loading: { shortLabel: '', label: localize('inlineSuggestionLoading', "Loading..."), loading: true, },
-				ghostText: { shortLabel: '$(lightbulb)', label: '$(copilot) ' + localize('inlineCompletionAvailable', "Inline completion available"), loading: false, },
-				inlineEdit: { shortLabel: '$(lightbulb-sparkle)', label: '$(copilot) ' + localize('inlineEditAvailable', "Inline edit available"), loading: false, },
-				noSuggestion: { shortLabel: '$(circle-slash)', label: '$(copilot) ' + localize('noInlineSuggestionAvailable', "No inline suggestion available"), loading: false, },
+				ghostText: { shortLabel: '$(lightbulb)', label: localize('inlineCompletionAvailable', "Inline completion available"), loading: false, },
+				inlineEdit: { shortLabel: '$(lightbulb-sparkle)', label: localize('inlineEditAvailable', "Inline edit available"), loading: false, },
+				noSuggestion: { shortLabel: '$(circle-slash)', label: localize('noInlineSuggestionAvailable', "No inline suggestion available"), loading: false, },
 			};
 
 			store.add(this._languageStatusService.addStatus({
